test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -139,6 +139,17 @@
                    :certname "foo.local"}}
                 (set result))))))))
 
+(deftest query-with-explain
+  (svc-utils/with-single-quiet-pdb-instance
+    (let [query-fn (partial query (get-service svc-utils/*server* :PuppetDBServer))]
+
+      (check-service-query
+       :v4 ["from" "facts" ["=" "certname" "foo.local"]]
+       {:explain true}
```

Comment:
Looking at this more closely it's only a testing issue and I don't think we need to update the test helper. The `check-service-query` func calls `(realized? result-set)` and the `result-set` is an empty vec when nothing is returned instead of the lazy-seq that's returned where there are results.